openai_agent/send_email.py

```diff
@@ -0,0 +1,76 @@
+import http.client
+import json
+from codecs import encode
+from os import environ
+from typing import Any, Dict
+
+INFOBIP_API_KEY = environ["INFOBIP_API_KEY"]
+INFOBIP_BASE_URL = environ["INFOBIP_BASE_URL"]
+
+
+def send_email(
+    *, from_email: str = "info@airt.ai", to_email: str, subject: str, body_text: str
```

Comment:
Please check with Harish which email address should we use.

info@captn.ai ?

Reply:
support@captn.ai, It is not created yet though